src/peep/staff.c

```diff
@@ -503,7 +503,9 @@ uint16 hire_new_staff_member(uint8 staffType)
 	int eax, ebx, ecx, edx, esi, edi, ebp;
 	ecx = edx = esi = edi = ebp = 0;
 	eax = 0x8000;
-	ebx = staffType << 8 | GAME_COMMAND_FLAG_APPLY;
+
```

Comment:
Since your editing this function you might as well refactor the names. `eax = command_x = 0x8000`, `ebx` keep the same, `ecx = command_y = 0`, `edx = command_z = 0`, `edi = new_sprite_index = 0`, `ebp` keep the same.